### ISSUE TITLE
Semantic versioning

### ISSUE THREAD

**Issue by anowacki** [mantle-convection-constrained/terratools#149]
@jamespanton93 I notice that in a recent patch release (v1.0.1 –> v1.0.2) we added a new feature.  semver.org suggests that patch bumps are for bugfixes, while minor version increments are for new features.  (Major version bumps are for incompatible changes.)  I have mentioned this in [CONTRIBUTING.md](https://github.com/mantle-convection-constrained/terratools/blob/main/CONTRIBUTING.md#contributing-code); so really we should be on v1.1.0 now.  Do we think this is enough for the devdocs, or should we add more about this?

**Comment by jamespanton93** [comment]
Yes you are correct. Perhaps the only thing to add is what sort of changes warrant a patch release. Perhaps Do you think we should retrospectively move to v.1.1.0 now? I should also add an example of how to use the new feature.

**Comment by anowacki** [comment]
At this stage I think we can leave it and not release another version, but it would be good to ensure we follow our policies in future.

If you think a bit more devdoc discussion on semver is needed beyond what's there, then feel free to add something.  We can leave this open for that in the meantime.
